watson_developer_cloud/speech_to_text_v1.py

```diff
@@ -61,6 +130,64 @@ def recognize(self, audio, content_type, continuous=False, model=None,
                             data=audio, params=params,
                             stream=True, accept_json=True)
 
+    def recognize_ws(self, audio, content_type,
```

Comment:
We should return an object with a `close()` method so that the user can close the WS